src/HttpServer.cc

```diff
@@ -586,7 +586,7 @@ void Http2Handler::start_settings_timer() {
 
 int Http2Handler::fill_wb() {
   if (data_pending_) {
-    auto n = std::min(wb_.wleft(), data_pendinglen_);
+    auto n = (std::min)(wb_.wleft(), data_pendinglen_);
```

Comment:
Since the MS defined the min and max macros so the std::min and std::max can't works with VS.
Please see this link: https://stackoverflow.com/questions/2789481/problem-calling-stdmax